docker/solr/Dockerfile

```diff
@@ -0,0 +1,123 @@
+FROM openjdk:11-jre
```

Comment:
Dockerfile should be generated only in `docker/solr/8/debian10/8.6/Dockerfile`
I believe this top level Dockerfile is redundant?

Reply:
What's this file needed for?